Makefile

```diff
@@ -571,8 +579,12 @@ docs/fields.md: api/openapi-spec/swagger.json $(shell find examples -type f) hac
 	env ARGO_SECURE=false ARGO_INSECURE_SKIP_VERIFY=false ARGO_SERVER= ARGO_INSTANCEID= go run ./hack docgen
 
 # generates several other files
-docs/cli/argo.md: $(CLI_PKGS) server/static/files.go hack/cli/main.go
+docs/cli/argo.md: $(CLI_PKGS) hack/cli/main.go
+    # If the static file does not exist, generate a mock file with required function signatures
+	[ -e ./server/static/files.go ] || echo -e "//DELETEME\npackage static\nimport \"net/http\"\nfunc ServeHTTP(w http.ResponseWriter, r *http.Request) {}\nfunc Hash(file string) string { return \"\" }" > ./server/static/files.go
```

Comment:
Nifty trick to save the need to spend 4 min generating static files JUST for these two function signatures

Reply:
can we use this everywhere?

Reply:
maybe create a stub file we can use?

./server/static/files.go.stub

Reply:
Yup, will rework